mulesoft/docs-site-ui
# [W-10592849] banner for beta

ref: W-10592849

Add a banner for beta only.

beta:
<img width="500" alt="Screen Shot 2022-07-15 at 1 30 30 PM" src="https://user-images.githubusercontent.com/10934908/179313795-b4364c44-b182-4d00-95eb-de97d6196424.png">

archive:
<img width="500" alt="Screen Shot 2022-07-15 at 1 30 34 PM" src="https://user-images.githubusercontent.com/10934908/179313798-2ec0e658-14a8-4ac8-958d-d8425f63cb6b.png">

To test:
1. run `gulp bundle` to create a local UI bundle
2. open antora-site-playbook and switch to the beta branch
3. select one of the playbook, point the UI bundle to the local ui-bundle.zip path, run `make gen-site playbook=replace_me_with_the_playbook_path`
4. verify that the site has the beta site banner (see first screenshot)
5. revert the change, and switch to the archive branch
6. open antora-playbook.yml (or create a my-playbook.yml). Point the UI bundle to the local ui-bundle.zip path, run `make gen-site playbook=replace_me_with_the_playbook_path`
7. verify that the site has the archive site banner (see second screenshot)
